test/vision-model-checker.test.ts

```
@@ -44,24 +45,26 @@ describe("isVisionModel", () => {
+    expect(isVisionModel("another-vision-model", customVisionModels)).toBe(true);
+    expect(isVisionModel("custom-vlm", customVisionModels)).toBe(true);
+    expect(isVisionModel("another-vlm", customVisionModels)).toBe(true);
+    expect(isVisionModel("unrelated-model", customVisionModels)).toBe(false);
   });
 
   test("should handle empty or missing VISION_MODELS", () => {
     process.env.VISION_MODELS = "";
-    expect(isVisionModel("unrelated-model")).toBe(false);
+    expect(isVisionModel("unrelated-model", customVisionModels)).toBe(false);
 
     delete process.env.VISION_MODELS;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)